__tests__/api.test.js

```diff
@@ -201,3 +201,67 @@ describe("6. GET /api/articles", () => {
       });
   });
 });
+
+describe("11. GET /api/articles queries", () => {
+  test("should return status 200", () => {
+    return request(app).get("/api/articles?sort_by=article_id").expect(200);
+  });
+  test("should return articles sorted by article_id when passed article id as query", () => {
```

Comment:
This is a little too specific in the description. We're not testing specifically for article_id, but that a sort_by query in general would work